<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>740</width>
      <height>550</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Params">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QProgressBar" name="leftMotorSpeed">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>50</width>
        <height>320</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="rightMotorSpeed">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>50</width>
        <height>320</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QDial" name="dial_Inclination">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
     </widget>
     <widget class="QDial" name="dial_Giros">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>220</width>
        <height>220</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdOptSup_1">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdOptSup_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>340</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdOptSup_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>290</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdOptSup_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>340</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdVelocity">
      <property name="geometry">
       <rect>
        <x>399</x>
        <y>330</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Graphs">
     <attribute name="title">
      <string>Graphs</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Debug">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>220</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextRX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>220</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>220</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TX</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextTX">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>220</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDebugSendData">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>DEBUG</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextLeftMotor">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>70</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextRightMotor">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxData">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Motors</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>80</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>120</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Display text</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextDisplay">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>170</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_Clean">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>470</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>CLEAR WINDOWS</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR1">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>230</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IRs</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>290</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>320</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>350</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>380</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR7">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>410</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_IR8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>440</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonOpenConfig">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSerialConected">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey; border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelWifiConected">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: grey; border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
